doc/acknowledgments/src/notices-and-acknowledgments/other-software-included-or-referenced.tex

```diff
@@ -128,7 +128,7 @@ \subsection{MODELISAR Notice}\label{modelisar-notice}
 
 This file is licensed by the copyright holders under the BSD License\footnote{\url{http://www.opensource.org/licenses/bsd-license.html\textgreater}}:
 
-\begin{center}\rule{0.5\linewidth}{\linethickness}\end{center}
+\begin{center}\rule{0.5\linewidth}{0.4pt}\end{center}
```

Comment:
What's the purpose of this change?

Reply:
This is a fix for a warning generated by LaTeX. `\linethickness` is not a valid option for the second argument to `\rule` but the LaTeX compiler typically muddles through. Just fixing in the name of removing warnings from the LaTeX compiler.